### PR TITLE
Add some troubleshooting data to the chat reference

### PR DESCRIPTION
**Pull request by ProbablePrime**
It turns out that UNOTFOUND is very unhelpful, I've added a troubleshooting guide for how to handle that in the efforts it might help save the 40 minutes it usually takes to figure this one out.

### CODE REVIEW

File: src/reference/chat/index.pug [hunk 1]
```diff
@@ -163,4 +156,15 @@ block reference
...
             +highlightStr('json', JSON.stringify(data.example, null, 3))
+    h1(id='chat__troubleshooting') Troubleshooting
+    h3 Getting #[code UNOTFOUND] a lot?
+    p This error means that the chat server you're connecting to can't find a record of you when you try to authenticate. This can be caused by a number of reasons so make sure to check the following things.
+    ul
+        li That the channel you're trying to connect to is valid.
+        li That you've retrieved an authentication key from the #[a(href='/rest.html#chats__chat__get') GET /chats/{channelId}] endpoint recently. Authentication keys expire after a short period.
+        li That you're calling the auth method with 3 arguments that are not null in the following order:
+            ol
+                li The #[strong channel id] of the channel you wish to chat in.
```
Comment (JamyDev) [inline src/reference/chat/index.pug]: Maybe mention that these should be numbers in json?

File: src/reference/chat/index.pug [hunk 2]
```diff
@@ -163,4 +156,15 @@ block reference
                 +highlightStr('json', JSON.stringify(example.data, null, 3))
         else
             +highlightStr('json', JSON.stringify(data.example, null, 3))
+    h1(id='chat__troubleshooting') Troubleshooting
+    h3 Getting #[code UNOTFOUND] a lot?
+    p This error means that the chat server you're connecting to can't find a record of you when you try to authenticate. This can be caused by a number of reasons so make sure to check the following things.
+    ul
+        li That the channel you're trying to connect to is valid.
```
Comment (JamyDev) [inline src/reference/chat/index.pug]: Add in brackets `Make sure you're using the channelId and not the userId`

File: src/reference/chat/index.pug [hunk 3]
```diff
@@ -163,4 +156,15 @@ block reference
...
+        li That you've retrieved an authentication key from the #[a(href='/rest.html#chats__chat__get') GET /chats/{channelId}] endpoint recently. Authentication keys expire after a short period.
```
Comment (connor4312) [inline src/reference/chat/index.pug]: > That you've retrieved an authentication key from the #[a(href='/rest.html#chats__chat__get') GET /chats/{channelId}] endpoint recently

Should be the opposite

Comment (connor4312) [inline src/reference/chat/index.pug]: ah my bad can't read

Comment (ProbablePrime) [inline src/reference/chat/index.pug]: So we're both potato :D

Comment (JamyDev) [inline src/reference/chat/index.pug]: 🍠

**Pull request merged**